app/api/user_favourite_events.py

```diff
@@ -94,3 +98,25 @@ class UserFavouriteEventRelationship(ResourceRelationship):
     methods = ['GET']
     data_layer = {'session': db.session,
                   'model': UserFavouriteEvent}
+
+
+@favourite_event_routes.route('/user-favourite-events-by-event-id/<identifier>', methods=['DELETE'])
+@jwt_required()
+def delete_favourite_by_event_id(identifier):
+    id = 'identifier'
+
+    if identifier.isdigit():
+        id = 'id'
+
+    event = safe_query(db, Event, id, identifier, 'event_' + id)
+
+    user_favourite_event = UserFavouriteEvent.query.filter_by(user=current_user, event_id=event.id).first()
+    if user_favourite_event:
+        db.session.delete(user_favourite_event)
+        db.session.commit()
+    else:
+        raise ConflictException({'pointer': '/data/relationships/event'}, "Event is not favourited")
```

Comment:
Why ConflictException?

Reply:
@iamareebjamal Corrected it. Please review.